packages/api/config/workflowLambdas.yml

```diff
@@ -0,0 +1,11 @@
+sf2snsStart:
```

Comment:
There is a very detailed PR description but I'm not sure I understand why we are separating out workflowLambdas

Reply:
It's a way to easily identify/differentiate which API package lambdas are intended to be used in workflows.

Reply:
I see the reasoning but seems like an unnecessary change for this PR - it's just an organizational thing if I understand correctly and I'm not sure it adds much value. I would prefer this change reverted but also non-blocking if you leave as-is

Reply:
It's not quite just an organizational change, we need to have some internal way to differentiate between lambdas we want to version and lambdas we don't, as there is conditional logic that acts on those lists in different ways.       

This, however, highlights a different issue, namely that we're not going to process the excluded lambdas, as the version template/kes code will not run process on them.  I missed this because I didn't test an API lambda change, so the correct build files were already out on s3     I'm going to push up a fix for that shortly.

I think it would be possible to instead address this concern by adopting a version exclusion flag on a per-lambda basis in the configuration, but that adds more configuration possibilities.   🤔